src/script/components/InputBar/TypingIndicator/TypingIndicator.tsx

```diff
@@ -0,0 +1,167 @@
+/*
```

Comment:
Let's create an `index.ts` that exports this component and the state (so that it's easier to import from somewhere else)